Mathlib/NumberTheory/NumberField/Discriminant.lean

```diff
@@ -41,6 +43,43 @@ theorem discr_eq_discr {ι : Type*} [Fintype ι] [DecidableEq ι] (b : Basis ι
   let b₀ := Basis.reindex (RingOfIntegers.basis K) (Basis.indexEquiv (RingOfIntegers.basis K) b)
   rw [Algebra.discr_eq_discr (𝓞 K) b b₀, Basis.coe_reindex, Algebra.discr_reindex]
 
+open MeasureTheory MeasureTheory.Measure Zspan NumberField.mixedEmbedding
+  NumberField.InfinitePlace ENNReal NNReal Complex
+
+theorem _root_.NumberField.mixedEmbedding.volume_fundamentalDomain_latticeBasis :
+    volume (fundamentalDomain (latticeBasis K)) =
+      (2 : ℝ≥0∞)⁻¹ ^ Fintype.card { w : InfinitePlace K // IsComplex w } * sqrt ‖discr K‖₊ := by
```

Comment:
Should `Fintype.card { w : InfinitePlace K // IsComplex w }` get a name? It would improve readability of this statement. Maybe
```
abbrev NrComplexPlaces := Fintype.card { w : InfinitePlace K // IsComplex w }
```
or something like that?

Reply:
That's actually a very good idea since these numbers show up at many places and will show up even more in forthcoming PRs. I was almost tempted to call them r₁ and r₂ but I guess your suggestion `NrRealPlaces` and `NrComplexPlaces` is more explicit.